Headers/CoreGraphics/CGAffineTransform.h

```diff
@@ -212,22 +212,28 @@ GS_AFTR_SCOPE CGAffineTransform CGAffineTransformRotate(
 
 GS_AFTR_SCOPE bool CGAffineTransformEqualToTransform(CGAffineTransform t1, CGAffineTransform t2)
 {
-  if (t1.a != t2.a) {
+  if (t1.a != t2.a)
+  {
     return false;
   }
-  if (t1.b != t2.b) {
+  if (t1.b != t2.b)
+  {
```

Comment:
GNU style, the way I understand it, would require two additional spaces before the brace. For example:

````
if (t1.b != t2.b)
  {
    return false;
  }
```